### PR TITLE
Task A: Implemented redirect route for task console to get better UX for users

### PR DESCRIPTION
**Pull request by andriyun**
Custom path: `/execute-foloeb-task`

The ability to refer to the specific task in email is possible via token, for example 
`/execute-foloeb-task?token=XebF7BbNosDZNM-GNDP7ADH0vaPbR74pLSf8pnXCalE`

The code is available for testing as a composer package https://packagist.org/packages/os2forms/os2forms_forloeb#dev-AARHUS-ITK-A

### CODE REVIEW

File: src/ForloebTaskConsole.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      $sid = $engine->getEntityIdentiferByUniqueID($processID, $task['data']['inherit_webform_unique_id'] ?? '');
+      $webform_submission = $sid ? WebformSubmission::load($sid) : NULL;
+
+      if ($webform_submission && $webform_submission->getToken() != $token) {
+        return $queueRecord;
+      }
+    }
+
+    return NULL;
+  }
```
Comment (rimi-itk) [inline src/ForloebTaskConsole.php:64]: Is it possible to get the queue record directly from the token without having to load and iterate over all queues and submissions, i.e. something like

```
// Warning: Pseudocode ahead!
$submission = $this->entityTypeManager->getStorage('webform_submission')->findByToken($token);
return NULL !== $submission
  ? $this->entityTypeManager->getStorage('maestro_queue')->findBySubmission($submission)
  : NULL;

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks very good! Nice and clean solution.

Some comments and suggestions added.

File: os2forms_forloeb.routing.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+os2forms_forloeb.forloeb_task_console_controller_execute:
+  path: 'execute-foloeb-task'
```
[the remark this answers is not in the document]
Comment (andriyun) [reply os2forms_forloeb.routing.yml]: Fixed

File: src/Controller/ForloebTaskConsoleController.php [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+          break;
+
+        case 'function':
+          $redirect_to = Url::fromRoute('maestro.execute', $query_options);
+          break;
+      }
+    }
+    else {
+      $this->messenger()->addWarning($this->t('No tasks found to execute.'));
+    }
```
[the remark this answers is not in the document]
Comment (andriyun) [reply src/Controller/ForloebTaskConsoleController.php]: Added early return

File: src/ForloebTaskConsole.php (hunk 1 above)
Comment (andriyun) [reply src/ForloebTaskConsole.php]: I don't think it's possible in an easy way. I can not find a direct connection between `maestro_queue` and `webform_submission` to implement this kind of lookup.

By idea, `maestro_queue` can have different handlers and reference to webform submission is on of them.

**Comment by andriyun** [comment]
Hi @rimi-itk 
Unfortunately, I can not merge this PR to develop branch.
Maybe you have access, and can do this for me?

**Comment by rimi-itk** [comment]
Sorry, @andriyun, I don't have permissions to merge, but maybe @madsnorgaard or @agger-magenta can help us?

**Comment by agger-magenta** [comment]
It's strange, but I can't seem to merge this either. @madsnorgaard, can you do it?

**Comment by madsnorgaard** [comment]
Squashed and merged.

File: src/ForloebTaskConsole.php (hunk 1 above)
Comment (andriyun) [reply src/ForloebTaskConsole.php]: I finally found a way to get it https://github.com/OS2Forms/os2forms_forloeb/pull/131/commits/f92e668e5af4671df9666a9e9928b6a7a47806dc

**Pull request merged**